notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,11 +15,9 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        result = subprocess.run([f"exe/STEMMUS_SCOPE {path_to_config}"], cwd = path_to_model, shell=True)
</pre><p>lets use absolute path for exe file as:</p><p><code>path_to_exefile = "STEMMUS_SCOPE/exe"</code></p><p><code>&nbsp;result = subprocess.run([f"STEMMUS_SCOPE {path_to_config}"], cwd = path_to_exefile, shell=True)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>